src/runtime/composables-v3/useStrapi.ts

```diff
@@ -0,0 +1,18 @@
+import type { Strapi3RequestParams } from '../types/v3'
+import { useStrapi3 } from '#imports'
+
+interface StrapiV3Client<T> {
+  count(contentType: string, params?: Strapi3RequestParams): Promise<number>
+  find<F = T[]>(contentType: string, params?: Strapi3RequestParams): Promise<F>
+  findOne<F = T>(contentType: string, id: string | number, params?: Strapi3RequestParams): Promise<F>
+  create<F = T>(contentType: string, data: Partial<F>): Promise<F>
+  update<F = T>(contentType: string, data: Partial<F>): Promise<F>
+  delete<F = T>(contentType: string, id: string | number | Partial<F>, data?: Partial<F>): Promise<F>
+}
+
+/**
+ * @deprecated use `useStrapi` for correct types
```

Comment:
Why is there a deprecated here?

Reply:
`useStrapi` calls this function, just with the right types. So using it directly, with typescript, is going to be buggy for people

Reply:
Sorry, I might be misunderstanding but isn't this `useStrapi`?

Reply:
Indeed I think it’s not needed here